azure_test.go

```diff
@@ -7,6 +7,7 @@ import (
 	"fmt"
 	"strings"
 
+	"github.com/Azure/azure-sdk-for-go/profiles/latest/msi/mgmt/msi"
```

Comment:
We should probably change this to use `/services` as we did in other places.

Reply:
Might want to run `go mod tidy` after that as well.

Reply:
Done